### ISSUE TITLE
Libguides: Table cell missing context

### ISSUE THREAD

**Issue by rschwab** [UCSCLibrary/SiteImprove#54]
Data tables include header and data cells arranged in a grid.

Each data cell <td> should be assigned to at least one column or row header cell <th>. When this is done, a screen reader can announce each data cell in context.

[Siteimprove link](https://my2.siteimprove.com/Accessibility/624596/77184128804/NextGen/Issue/1?conformance=&siteTargetIssueKinds=1,2&wcagVersion=21&pageSegments=&ruleName=sia-r77&ruleId=77&issueKind=1&tagFilterType=2&siteTarget.wcagVersion=21&siteTarget.conformanceLevels=0,1,3,4&siteTarget.issueKinds=1,2)

- [x] https://guides.library.ucsc.edu/c.php?g=666699&p=4689078
- [x] https://guides.library.ucsc.edu/contactus-old

![Image](https://github.com/user-attachments/assets/6186d41e-13db-4f35-aa8e-f9dd52091762)

- [ ] https://guides.library.ucsc.edu/discussions.php

![Image](https://github.com/user-attachments/assets/518c5bf7-0828-4b6e-afd9-a6bc3b4fa191)

**Comment by JessWaggs** [comment]
The only page left with this issue is the "Public Discussion" board which is system-generated. Per Springshare email on 7/10/25, we cannot hide, remove, or edit this page.
